mention "backfill" in the doc (#2657)
risingwavelabs/risingwave-docs: docs/sql/commands/sql-create-mv.md

```

:::tip

> `CREATE MATERIALIZED VIEW` will first **backfill** historical data from the referenced relations, and it may take a long time to complete depending on how much data needs to be backfilled.

Reviewer: ```suggestion
`CREATE MATERIALIZED VIEW` will first **backfill** historical data from the referenced relations, and completion time varies based on the volume of data to be backfilled.
```
